rules/box_rules/box_item_shared_externally.yml

```diff
@@ -8,6 +8,7 @@ LogTypes:
 Tags:
   - Box
   - Exfiltration:Exfiltration Over Web Service
+  - Configuration Required
```

Comment:
No action needed: 
panther_box_helpers require configuration. this detection doesn't directly require config, and the helper it imports does